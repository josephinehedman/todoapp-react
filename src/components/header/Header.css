.header {
    margin-top: 2rem;
    text-align: center;
    color: rgb(8, 0, 77);
  }

  @media all and (min-width: 560px) {
    .header
    {
      font-size: 1.5rem;
    }
  }  