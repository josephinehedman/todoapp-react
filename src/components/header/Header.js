import './Header.css';

const Header = () => {
    return (
        <header className="header">
            <h1 className="header__heading">
            Todo List
            </h1>
        </header>
    )
}

export default Header;