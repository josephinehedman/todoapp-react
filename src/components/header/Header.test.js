import renderer from 'react-test-renderer';
import Header from './Header'

describe('Header component', () => {
    test('should render the heading (by matching snapshot)', () => {
        const header = renderer.create(<Header />).toJSON();
        expect(header).toMatchSnapshot();
    });
});