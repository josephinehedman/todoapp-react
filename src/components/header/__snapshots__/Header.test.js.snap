// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render the heading (by matching snapshot) 1`] = `
<header
  className="header"
>
  <h1
    className="header__heading"
  >
    Todo List
  </h1>
</header>
`;
